lib/data_hygiene/title_fetcher.rb

```diff
@@ -0,0 +1,74 @@
+module DataHygiene
+  class TitleFetcher
```

Comment:
I considered that, but thought it sounded like it would update all of our titles, overwriting them in our database if they're different in GovDelivery, whereas this only sets titles on lists which don't have one and logs everything else. "Fetch" also makes it clearer that it's getting them from somewhere else as opposed to modifying them based on something local. I can change it if you like?